static/src/javascripts/projects/common/modules/experiments/tests/contributions-embed.js

```diff
@@ -0,0 +1,95 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-embed.html',
+    'common/utils/robust'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEmbed,
+             robust
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsEmbed20160822';
+        this.start = '2016-08-23';
+        this.expiry = '2016-08-27';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test contributions embed with amount picker.';
+        this.showForSensitive = false;
+        this.audience = 0.13;
```

Comment:
Asias response : ""If you are running a split a / b test -- which means showing the message to 50% and then donatum plus same message to the other 50% -- and you expect the component to produce about 3x increase over current approach, then expose each variant to 600,000 -- so, 1.2m total""

I think that this means we should have and equal (but distinct) audience share from the article test, hence the 0.13 offset and 0.13 audience